### ISSUE TITLE
Categories are lost when setting up new budget

### ISSUE THREAD

**Issue by rohieb** [danilkinkin/buckwheat#117]
Dear maintainer,

I've used the app for a month now and my budget ran until the 30th of June, so at the 1st of July I was prompted to set up a new budget, which I did. I exported the old spendings to CSV. However, after setting up the new budget (with the same numbers), when entering a new posting, the list of categories was empty, although I had used a lot of categories in the previous budget.

**Expected behavior**
I expected the categories to stay in place because I want to re-use them for the new budget.

**Screenshots**
Statistics screen before exporting the old budget:
![Screenshot_20240701-180005_](https://github.com/danilkinkin/buckwheat/assets/192014/91ca2c87-7e6d-4fcb-babc-64dbcdaae88f)

Entering a new posting after setting up a new budget (two categories have already been entered again):
![Screenshot_20240703-102905](https://github.com/danilkinkin/buckwheat/assets/192014/2b6548ee-55df-4d01-a8ca-72fce1a4b0e9)

**More Info**
 - Device: Pixel 6a
 - OS: Android 14
 - Buckwheat Version: 4.7.0 from F-Droid

**Comment by danilkinkin** [comment]
Yep, nowadays, it's just suggestions based on your active budget. When I making it I think it will be more correct (I want in future add multi budgets feature), but now I'm facing with same inconvenience. 

Maybe you are right, maybe better to save suggestion all time

**Comment by danilkinkin** [comment]
Additional point from #112 by @Bugsbane
> An option at the end of the budget period to keep tags or not. This would add to extra options unnecessarily though.
Another option would be a "Tag editor" section of the app where you could rename and delete tags. This seems unnecessarily complicated though for what's needed.
